files/mix_class/app-template.service

```diff
@@ -1,14 +0,0 @@
-[Unit]
-Description=@@DESCRIPTION@@
-
-[Service]
-
-Environment="HOME=@@APP_HOME@@"
-Environment="USER_CONFIG_FILE=@@USER_CONFIG_FILE@@"
-Environment="MUTABLE_DIR=@@MUTABLE_DIR@@"
```

Comment:
RELEASE_MUTABLE_DIR and HOME will now be set in the wrapper. This is because e.g. 'systemctl start app@default' will be mapped to '/run/app@<latest_version>-default' and both variables need to be altered to that path anyway.